src/detections/filters/accordion-details/cameras/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { FilterWrapper } from 'detections/filters/accordion-details/filter-wrapper'
+import { RemoveButton } from 'detections/filters/accordion-details/remove-button'
+import { FilterTitle } from 'detections/filters/accordion-details/filter-title'
+import { Content } from 'detections/filters/accordion-details/cameras/content'
+
+export const Cameras = index => <FilterWrapper>
```

Comment:
Let's make sure we pass props the way props are meant to be passed: as fields inside of an object.

This applies to all files touched in this pr.

Reply:
done